crates/gen-host-js/src/lib.rs

```diff
@@ -48,35 +51,35 @@ struct Js {
 pub struct Opts {
     /// Disables generation of `*.d.ts` files and instead only generates `*.js`
     /// source files.
-    #[cfg_attr(feature = "clap", arg(long = "no-typescript"))]
+    #[cfg_attr(feature = "clap", arg(long))]
     pub no_typescript: bool,
     /// Provide a custom JS instantiation API for the component instead
     /// of the direct importable native ESM output.
     #[cfg_attr(
         feature = "clap",
         arg(
-            long = "instantiation",
+            long,
             short = 'I',
             conflicts_with = "compatibility",
             conflicts_with = "compat"
         )
     )]
     pub instantiation: bool,
+    /// Inline core WebAssembly modules as base64 strings.
+    /// Not recommended!
+    #[cfg_attr(feature = "clap", arg(long, hide = true))]
+    pub base64: bool,
```

Comment:
I personally think it's ok to leave this here by default, I don't think we need to go out of our way to hide it. It's useful for at least one case so if it's there for that seems like others may find it useful too and it may as well not be proactively hidden.

Reply:
Ok, sure.